docs/README.md

```diff
@@ -69,6 +69,14 @@ and Direnv installed.
 This folder contains another `.envrc` file that adds all the binaries needed to
 build the docs to `$PATH`.
 
+In short, when you `cd` into this folder, you should see this message:
+
+```sh
+direnv: error wire-server/docs/.envrc is blocked. Run `direnv allow` to approve its content
+```
+
+Run `direnv allow` to allow the `.envrc` file to modify your environment, and you should then have all the binaries/correct environment needed to build the docs.
```

Comment:
```suggestion
Run `direnv allow` to allow the `.envrc` file to modify your environment. Then, you should have everything (binaries, environment variables) needed to build the docs.
```

Reply:
Should we mention cachix here as well? Without it set up, `direnv allow` will take hours to complete.

Reply:
@elland do you have a link/recommendation of what to say about cachix?

Reply:
It's mentioned here https://github.com/wireapp/wire-server/blob/develop/docs/src/how-to/install/dependencies.rst :)